packages/client/hmi-client/src/components/workflow/ops/model-edit/tera-model-edit-node.vue

```diff
@@ -28,12 +28,21 @@ const props = defineProps<{
 }>();
 
 const model = ref<Model | null>(null);
+
 const updateModel = async () => {
 	const modelId = operator.getActiveOutput(props.node)?.value?.[0];
 	if (modelId && modelId !== model?.value?.id) {
 		model.value = await getModel(modelId);
 	}
 };
+
 onMounted(updateModel);
-onUpdated(updateModel);
+
+// Updates output selection
+watch(
+	() => props.node.active,
+	async () => {
+		await updateModel;
```

Comment:
`await updateModel()`

Reply:
try this, you can remove the onMounted too
```
// Updates output selection
watch(
	() => props.node.active,
	() => {
		updateModel();
	},
	{ immediate: true }
);
```

Reply:
Let's rename this function please, update model sounds confusing when you are actually fetching the model